features/app-launch-handler.yml

```
@@ -0,0 +1,9 @@
+name: Launch Handler
+description: The `launch_handler` manifest member is used to configure how web app launches should behave with respect to already open web app instances.
+spec: https://wicg.github.io/web-app-launch/#dfn-launch_handler
```

Since this feature seems to have a one-to-mapping with the spec, I think we can just link to the top:

```suggestion
spec: https://wicg.github.io/web-app-launch/
```

---

Done!